### PR TITLE
Roadmap styles

### PR DESCRIPTION
**Pull request by johnkuney**

### CODE REVIEW

File: v2/src/pages/roadmap.js [hunk 1]
```diff
@@ -301,112 +347,113 @@ const RoadmapPage = () => {
...
-              <fbt desc="Roadmap's upgrade activation explaination">
-                Rather than a specific block height, the upgrades are based on a
-                timestamp. When the median timestamp of the most recent 11
-                blocks (MTP-11) is greater than or equal to UNIX timestamp of
-                November 15th (or May 15th) 12:00:00 UTC, the new protocol rules
-                take effect. This better enables your business to prepare for
-                the upgrade at a particular estimated time, plus or minus a few
-                hours.
+        <Roadmap />
+        <div className={roadmapStyle.topSection}>
```
Comment (deadalnix) [inline v2/src/pages/roadmap.js:388]: The topSection name here seems to be inappropriate.

File: v2/src/pages/roadmap.js [hunk 2]
```diff
@@ -254,41 +296,45 @@ const Roadmap = () => {
               <ion-icon class="planned" name="add-circle" />
               <fbt desc="Planed icon on the roadmap legend">Planned</fbt>
             </p>
-          </div>
+          </div> */}
```
Comment (deadalnix) [inline v2/src/pages/roadmap.js:299]: Do not leave commented out code in there.

File: v2/src/pages/roadmap.js [hunk 3]
```diff
@@ -236,16 +248,46 @@ const Roadmap = () => {
...
     <>
       <div className={roadmapStyle.roadmap}>
-        <div className={`${roadmapStyle.row} ${roadmapStyle.legend}`}>
-          <div className={roadmapStyle.complete}>
-            <p>
-              <ion-icon class="complete" name="checkmark-circle" />
+        <div className={roadmapStyle.toplegendcontainer}>
+          <div className={roadmapStyle.toplegendcol}>
+            <div className={roadmapStyle.completetop}>
+              {<Check />}
```
Comment (deadalnix) [inline v2/src/pages/roadmap.js:254]: No need for curly braces.

File: v2/src/pages/roadmap.js [hunk 4]
```diff
@@ -236,16 +248,46 @@ const Roadmap = () => {
...
+              <div className={roadmapStyle.completetoptext}>
+                <fbt desc="Complete icon on the roadmap legend">Complete</fbt>
+              </div>
+            </div>
+          </div>
+          <div className={roadmapStyle.toplegendcol}>
+            <div className={roadmapStyle.underwaytop}>
+              {<Underway />}
```
Comment (deadalnix) [inline v2/src/pages/roadmap.js:262]: dito

File: v2/src/pages/roadmap.js [hunk 5]
```diff
@@ -236,16 +248,46 @@ const Roadmap = () => {
...
+              <div>
+                <fbt desc="Underway icon on the roadmap legend">Underway</fbt>
+              </div>
+            </div>
+          </div>
+          <div className={roadmapStyle.toplegendcol}>
+            <div className={roadmapStyle.plannedtop}>
+              {<Planned />}
```
Comment (deadalnix) [inline v2/src/pages/roadmap.js:270]: dito

**Review by deadalnix — CHANGES_REQUESTED** [review]
Some small changes, but overall, LGTM

File: v2/src/assets/icons/planned.svg [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<svg
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 512 512"
+          class="s-ion-icon"
+        >
+          <path d="M256 48C141.125 48 48 141.125 48 256s93.125 208 208 208 208-93.125 208-208S370.875 48 256 48zm107 229h-86v86h-42v-86h-86v-42h86v-86h42v86h86v42z"></path>
+        </svg>
```
[the remark this answers is not in the document]
Comment (johnkuney) [reply v2/src/assets/icons/planned.svg]: Fixed

File: v2/src/pages/roadmap.js (hunk 1 above)
Comment (johnkuney) [reply v2/src/pages/roadmap.js]: renamed

File: v2/src/pages/roadmap.js (hunk 2 above)
Comment (johnkuney) [reply v2/src/pages/roadmap.js]: For sure removed it

File: v2/src/pages/roadmap.js (hunk 3 above)
Comment (johnkuney) [reply v2/src/pages/roadmap.js]: deleted

File: v2/src/pages/roadmap.js (hunk 4 above)
Comment (johnkuney) [reply v2/src/pages/roadmap.js]: same

File: v2/src/pages/roadmap.js (hunk 5 above)
Comment (johnkuney) [reply v2/src/pages/roadmap.js]: deleted

**Pull request merged**